Gopkg.toml

```diff
@@ -150,3 +150,8 @@ ignored = ["github.com/uber/cadence/.gen"]
 [[constraint]]
   name = "github.com/valyala/fastjson"
   version = "1.4.1"
+
```

Comment:
Extra enter.

Reply:
Fixed, somehow merging with master this automatically. Good find!